partcad-cli/src/partcad_cli/cli.py

```
@@ -117,105 +91,10 @@ def main():
-        elif args.command == "list-assemblies":
-            with pc.logging.Process("ListAssy", "this"):
-                cli_list_assemblies(args, ctx)
-
-        elif args.command == "render":
-            with pc.logging.Process("Render", "this"):
-                cli_render(args, ctx)
-
-        elif args.command == "inspect":
+        if args.command == "inspect":
             with pc.logging.Process("inspect", "this"):
                 cli_inspect(args, ctx)
```

Ruff (0.8.2)

`cli_inspect` may be undefined, or defined from star imports

(F405)